sycl/test-e2e/USM/hmemll.cpp

```diff
@@ -1,6 +1,9 @@
 // RUN: %{build} -o %t1.out
 // RUN: %{run} %t1.out
 
+// UNSUPPORTED: windows && arch-intel_gpu_bmg_g21 && (!build-mode && run-mode)
+// UNSUPPORTED-TRACKER: https://github.com/intel/llvm/issues/17582
```

Comment:
NIT: Maybe we can change the title of the GH issue? On the first glance, it looked unrelated to this test